applicationset/utils/utils.go

```diff
@@ -99,10 +123,15 @@ func (r *Render) deeplyReplace(copy, original reflect.Value, replaceMap map[stri
 			// specific case time
 			if currentType == "time.Time" {
 				copy.Field(i).Set(original.Field(i))
-			} else if currentType == "Raw.k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1" {
+				// Raw.k8s.io/apimachinery/pkg/runtime
```

Comment:
The comment does not give any explanation. Do we need this?

Reply:
I will delete it. It was a mistake

Reply:
Allo good!